md/getting_started.md

```diff
@@ -1,56 +1,48 @@
 # Getting started
 
-Bonita Intelligent Continuous Improvement installation and configuration instruction. 
+Bonita Intelligent Continuous Improvement Add-on installation and configuration instruction. 
 
-This chapter cover *Quick start* and *Production* installation.
+This chapter covers *Evaluation* and *Production*, two different installation modes.
 
 ## Download
 
-Download ICI from [Bonitasoft Customer Portal](https://customer.bonitasoft.com/).
+Download ICI Add-on from [Bonitasoft Customer Portal](https://customer.bonitasoft.com/).
 
 The archive contains:
-* The installation guide: `installation-guide.md`
+* An installation guide: `installation-guide.md`
 * The ICI application: `ici-application-<version>.zip`
-* The Living applications
+* Two Living Applications: Case Monitoring with predictions, and Configuration of the first application
 * An installer: `bin/bonita-ici`
 
 ## Installation
 
-Installation can be done in a *Quick start* or *Production* mode. 
- 
-* *Quick start* mode is designed to try out the ICI module on an existing Bonita platform.
-* *Production* mode should be used when using ICI module for production.
-
-## Pre-requisites
-
-* A Bonita platform
-* Java
-* Docker (for *Quick start*)
+Installation can be done in a *Evaluation* or *Production* mode. The major difference between those mode is that 
+Elasticsearch and ICI server are provided in Docker container for evaluation mode and don't allow to connect 
+to an existing Elasticsearch cluster.
 
 :::info
 Check all [pre-requisites](./prerequisites.md) prior to install.
 :::
 
-If Bonita engine database vendor is oracle, get JDBC driver jar from Oracle (not included in distribution).
+* *Evaluation mode* mode is designed to try out the ICI add-on on an existing Bonita platform.
+* *Production* mode should be used when using ICI add-on for production.
 
-### Quick start mode
+### Evaluation mode
 
 1. Ensure you have java installed, using command `java -version`
 2. Ensure you have Docker installed, using command `docker --version`
-3. Docker requires an active internet connection to pull Elasticsearch docker image and to build ICI application image
-4. Follow installation guide Quick start instructions from the archive.
+3. Docker requires an active Internet connection to pull Elasticsearch Docker image and to build ICI application image
+4. Follow the installation guide "Quick start" instructions from the archive.
```

Comment:
`Quick start` should be rename to `evaluation` as i did in installation guide?

Reply:
Everything that makes the whole procedure consistent is good.
So if this is evaluation, let's make it evaluation too, sure. Good catch.